cvat-ui/src/components/annotation-page/canvas/views/canvas2d/brush-tools.tsx

```diff
@@ -201,110 +258,131 @@ function BrushTools(): React.ReactPortal | null {
     }
 
     return ReactDOM.createPortal((
-        <div className='cvat-brush-tools-toolbox' style={{ top, left, display: visible ? '' : 'none' }}>
-            <Button
-                type='text'
-                className='cvat-brush-tools-finish'
-                icon={<Icon component={CheckIcon} />}
-                onClick={() => {
-                    if (canvasInstance instanceof Canvas) {
-                        if (editableState) {
-                            canvasInstance.edit({ enabled: false });
-                        } else {
-                            canvasInstance.draw({ enabled: false });
-                        }
-                    }
-                }}
+        <>
+            <GlobalHotKeys
```

Comment:
`I think you may easily put <GlobalHotKeys> into <div> and we do not need fragment in this case`